src/Symfony/Bridge/Twig/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.3.0
+-----
+
+* added a new `serialize` filter to serialize objects using the Serializer component.
```

Comment:
capitalize the line, do not end with a period ([ref]( https://symfony.com/doc/current/contributing/code/conventions.html#writing-a-changelog-entry))

Reply:
done